code/modules/client/preference/loadout/loadout_neck.dm

```diff
@@ -0,0 +1,104 @@
+/datum/gear/neck
+	subtype_path = /datum/gear/neck
+	slot = slot_neck
+	sort_category = "Neck"
+
+//Mantles
+/datum/gear/neck/mantle
+	display_name = "mantle"
+	path = /obj/item/clothing/neck/mantle
+
+/datum/gear/neck/old_scarf
+	display_name = "old scarf"
+	path = /obj/item/clothing/neck/mantle/old
+
+/datum/gear/neck/regal_shawl
+	display_name = "regal shawl"
+	path = /obj/item/clothing/neck/mantle/regal
+
+/datum/gear/neck/mantle/job
+	subtype_path = /datum/gear/neck/mantle/job
+	subtype_cost_overlap = FALSE
+
+/datum/gear/neck/mantle/job/captain
+	display_name = "mantle, captain"
+	path = /obj/item/clothing/neck/mantle/captain
+	allowed_roles = list("Captain")
+
+/datum/gear/neck/mantle/job/chief_engineer
+	display_name = "mantle, chief engineer"
+	path = /obj/item/clothing/neck/mantle/chief_engineer
+	allowed_roles = list("Chief Engineer")
+
+/datum/gear/neck/mantle/job/chief_medical_officer
+	display_name = "mantle, chief medical officer"
+	path = /obj/item/clothing/neck/mantle/chief_medical_officer
+	allowed_roles = list("Chief Medical Officer")
+
+/datum/gear/neck/mantle/job/head_of_security
+	display_name = "mantle, head of security"
+	path = /obj/item/clothing/neck/mantle/head_of_security
+	allowed_roles = list("Head of Security")
+
+/datum/gear/neck/mantle/job/head_of_personnel
+	display_name = "mantle, head of personnel"
+	path = /obj/item/clothing/neck/mantle/head_of_personnel
+	allowed_roles = list("Head of Personnel")
+
+/datum/gear/neck/mantle/job/research_director
+	display_name = "mantle, research director"
+	path = /obj/item/clothing/neck/mantle/research_director
+	allowed_roles = list("Research Director")
+
+//Cloaks
+/datum/gear/neck/cloak
+	display_name = "cloak, grey"
+	path = /obj/item/clothing/neck/cloak
```

Comment:
It's spelled "grey" here *(the correct way)*, but the actual item is called "gray cloak". One of them should probably be changed so that they're consistent.

Reply:
Renamed the cloak to "grey".